packages/web/src/features/landing/frames/NoticeSectionFrame.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import React from "react";
+import styled from "styled-components";
+import { NoticeInfo } from "@sparcs-clubs/web/features/notice/types/notice.type";
+import Link from "next/link";
+import paths from "@sparcs-clubs/web/constants/paths";
+import MoreSectionTitle from "../components/MoreSectionTitle";
+import NoticeCard from "../components/NoticeCard";
+
+const NoticeSectionFrameInner = styled.div`
+  display: flex;
```

Comment:
얘는 왜 필요할까요?? 한 번 생각해보세요ㅎㅎ

```suggestion
  flex: 1;
  display: flex;
```

Reply:
mazayo!